renku/core/models/refs.py

```diff
@@ -78,7 +78,9 @@ def delete(self):
 
     def set_reference(self, reference):
         """Set ourselves to the given reference path."""
-        reference_path = Path(reference).resolve().absolute()
+        reference_path = Path(
+            os.path.realpath(os.path.abspath(str(reference)))
+        )
```

Comment:
It does, but it's monkey patched for python 3.5 and I wanted to get rid of it since it's a very bad practice to have monkey patched things which people are not aware of. I'll revert it for now and create a ticket for it.

Reply:
#722 722